packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -1033,22 +1033,25 @@ private MethodBodyStatement[] CreateWritePropertiesStatements()
             for (var i = 0; i < propertyCount; i++)
             {
                 var property = _model.Properties[i];
-                var propertyWireInfo = property.WireInfo;
-                var propertySerializationName = propertyWireInfo?.SerializedName ?? property.Name;
-                var propertyMember = new MemberExpression(null, propertySerializationName);
```

Comment:
this is incorrect because if we want to get the member of "this property", why we get it from the serialized name instead of `property.Name`?